src/dra/play.c

```diff
@@ -24,7 +24,7 @@ u8 g_StageSelectOrder[] = {
     STAGE_RBO8,       STAGE_TOP_ALT,  STAGE_ENDING,
 };
 
-// The effect of this array is unknown
+// This is used for NO1 Weather
 // 0: 25% of chance
 // 1: 62.5% of chance
 // 2: 12.5% of chance
```

Comment:
Might be good to rename this variable to `g_RandomizeOuterWallWeather`. Think it's only used in this file so impact should be minimal

Reply:
Or I think your comment in `castle_flags.h` calls this `g_NO1WeatherOptions`.

Reply:
Makes sense to me.  I've been naming things by overlay for consistency, so I'll do that here as well.